processor/README.md

```diff
@@ -193,8 +193,31 @@ examples on using the processor.
 ## <a name="probabilistic_sampler"></a>Probabilistic Sampler Processor
 <FILL ME IN - I'M LONELY!>
 
-## <a name="queued"></a>Queued Processor
-<FILL ME IN - I'M LONELY!>
+## <a name="queued"></a>Queued Retry Processor
+The queued retry processor uses a bounded queue to relay trace data from the
+receiver or previous processor to the next processor.
+Received trace data is enqueued immediately if the queue is not full . At the
+same time, the processor has one or more workers which consume the trace
+data in the queue by sending them to the next processor. On failure of
+consuming the trace data, the processor retries after some backoff delay
```

Comment:
Unclear what "consuming" refers to. Maybe say instead: "If relaying the trace data to the next processor or exporter in the pipeline fails ..."

Reply:
Updated. Thanks.